src/components/Form.js

```diff
@@ -1,19 +1,32 @@
 import React from 'react';
 import './Form.css';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const Form = () => (
-  <div>
-    <h1>ADD NEW BOOK</h1>
-    <form className="form">
-      <input type="text" placeholder="Book Title" />
-      <select name="authors">
-        <option value="author1">Author 1</option>
-        <option value="author2">Author 2</option>
-        <option value="author3">Author 3</option>
-      </select>
-      <button type="submit">Add Book</button>
-    </form>
-  </div>
-);
+const Form = () => {
+  const dispatch = useDispatch();
+  const addNewBook = () => {
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    const books = {
+      id: Math.floor(Math.random() * 100),
+      title,
+      author,
+    };
+    dispatch(addBook(books));
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
+  };
+  return (
+    <div>
+      <h1>ADD NEW BOOK</h1>
+      <form className="form">
+        <input type="text" placeholder="Book Title" id="title" />
+        <input name="authors" type="text" placeholder="Author" id="author" />
```

Comment:
- [ ] Kindly make sure the local state with `title`, and `author` have their inputs that update the local state every time they update (`onChange`).